pybamm/solvers/casadi_solver.py

```diff
@@ -41,10 +42,13 @@ class CasadiSolver(pybamm.BaseSolver):
         specified by 'root_method' (e.g. "lm", "hybr", ...)
     root_tol : float, optional
         The tolerance for root-finding. Default is 1e-6.
+    max_step : float, optional
+        Maximum allowed step size. Default is np.inf, i.e., the step size is not
+        bounded and determined solely by the solver.
     max_step_decrease_count : float, optional
         The maximum number of times step size can be decreased before an error is
         raised. Default is 5.
-    dt_max : float, optional
+    dt_event : float, optional
```

Comment:
I agree `dt_event` is a better name for this. Can you make it so that the solver also still accepts `dt_max` and raises a `DeprecationWarning`?